AutoImplement/StringWriter.cs

```diff
@@ -17,7 +17,7 @@ public class StringWriter {
          }
       }
 
-      public IDisposable Indent() => new IndentationScope(this);
+      public IDisposable Scope => new IndentationScope(this);

```

Comment:
Does this being a property cause any weird behavior? I would think that weird scoping would occur if a breakpoint was tripped and the `StringWriter` was expanded in Visual Studio.

Since it has side effects, it should probably be a method.